### PR TITLE
Add Formbricks to list

### PR DESCRIPTION
**Pull request by jobenjada**
We need a new category for form and survey tools :)

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [X] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.

### CODE REVIEW

**Comment by KieranRobson** [comment]
> We need a new category for form and survey tools :) 

A new category can be suggested if 3 or more tools fit into it. If you make an issue suggesting this, it can definitely be discussed!

File: software/formbricks [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: "Formbricks"
+website_url: "https://formbricks.com"
+source_code_url: "https://formbricks.com/github"
+description: "Open Source Form and Survey Builder. Run TypeForm like surveys and in app surveys. Alternative to Qualtrics, SurveyMonkey, Qualaroo, Hotjar, JotForm, Sprig, Refiner."
```
Comment (KieranRobson) [inline software/formbricks:4]:
```suggestion
description: "Open Source Form and Survey Builder. Run TypeForm like surveys and in app surveys (alternative to Qualtrics, SurveyMonkey, Qualaroo, Hotjar, JotForm, Sprig, Refiner)."
```

File: software/formbricks [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: "Formbricks"
+website_url: "https://formbricks.com"
+source_code_url: "https://formbricks.com/github"
```
Comment (KieranRobson) [inline software/formbricks:3]:
```suggestion
source_code_url: "https://github.com/formbricks/formbricks"
```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
A couple standardisation tweaks to be made

**Comment by nodiscc** [comment]
No reply for a month, closing.

**Pull request closed without merging**